apps/nuxt3-ssr/components/ContactForm.vue

```diff
@@ -38,7 +36,7 @@ defineEmits(["submitForm"]);
         :placeholder="field.placeholder"
         :options="(field as ISelectFormField).options"
       />
-      <div class="pl-3" :class="{ 'text-red-500': field.hasError }">
+      <div class="pl-3" :class="{ 'text-invalid': field.hasError }">
```

Comment:
I like semantic styles. Should be a guideiline for our lib

Reply:
Yes, that's a good idea. In the next PR, I'll start writing this out in the README